pkg/wharfyml/visit/visit.go

```diff
@@ -21,13 +21,15 @@ var (
 	ErrTooManyDocs      = errors.New("only 1 document is allowed")
 )
 
+// String will try read this YAML node as a string.
```

Comment:
`sed -i 's/try read/try to read/g' ./pkg/wharfyml/visit/visit.go`

Reply:
Cmon man, you're making it too easy :P